### ISSUE TITLE
change UI for configuration vs available resources

### ISSUE THREAD

**Issue by medyagh** [kubernetes/minikube#7617]
> I don't fully understand why we need to show the word "Kubernetes" twice.
> 
> The end goal was to make the start output more similar for different drivers:
> 
> `🔥 Creating virtualbox VM (CPUs=2, Memory=2200MB, Disk=20000MB) ...`
> 
> `🔥 Creating docker container (CPUs=2, Memory=2200MB, Disk=20000MB) ...`

what do u think about using multi lines so we can still keep the availables (very useful to have a sense what kind of system user has:
🔥  Creating docker container
    ▪  CPUs=2 (2 available)
    ▪  Memory=4000MB (7964MB available) 
    ▪  Disk=20000MB (10000MB available)

_Originally posted by @medyagh in https://github.com/kubernetes/minikube/pull/7478#issuecomment-612361926_

**Comment by afbjorklund** [comment]
Currently we only post this information ("available", a.k.a. host info) for the `none` driver:

```
🤹  Running on localhost (CPUs=4, Memory=7800MB, Disk=138379MB) ...
ℹ️  OS release is Ubuntu 16.04.6 LTS
```

For other drivers we hide this information and just show the configured VM size instead.
Which we _are_ using as limits for the container too,  with the `--cpus` and `--memory` flags.

The main problem for the docker driver is that we don't **know** the disk size outside Linux
(The podman driver only works on Linux, if anything, so doesn't have the Mac and Win issue)

But we could bring back the previous code from 41dd96659a37fa6df3ee9356de6d71724b885c4c, and only show it on Linux ?

Basically we show the statfs of the storage root directory, like `sudo df -h /var/lib/docker`

**Comment by tstromberg** [comment]
The guiding principles for minikube UX so far have been:

- no more than 1-line per 5-seconds of runtime
- output should fulfill a distinct user need, in priority of:
  - inform the user of what the next pause is for
  - inform the user of configuration choices made
  - provide breadcrumbs for triage (OS, version, runtime, warnings)

In my experience triaging 1600+ issues, I have not found available resources to be useful, so I don't feel like it meets the latter requirement. Likewise, the host info for the none driver is duplicate of what is already shown elsewhere.

I sent this doc for comments a few weeks ago, but it's worth revisiting in light of this issue: https://docs.google.com/document/d/1v_ueA1JeaXDIbb7XUWezcZ0hpXaOGlveMyj94es3DWM/edit#

# Single node 

```
😄  minikube v1.9.2 on Darwin 10.15.4
🔥  Creating a Docker container with 2 CPUs and 2200MB of RAM ...
🐳  Pulling Kubernetes v1.18.0 images for Docker 19.03.2 ...
☸️  Initializing control-plane ....
🎁  Enabling: default-storageclass, storage-provisioner ...
🏄  Done! kubectl is now configured to use "minikube"
```

With an average runtime approaching 30s - we get about 6 lines to work with. 

# Multi-node

```
😄  minikube v1.9.2 on Darwin 10.15.4
🔥  Creating Docker containers with 2 CPUs and 2200MB of RAM ...
🐳  Pulling Kubernetes v1.18.0 images for Docker 19.03.2 ...
☸️  Initializing control-plane ...
👪  Joining 2 worker nodes ...
🎁  Enabling: default-storageclass, storage-provisioner ...
🏄  Done! kubectl is now configured to use "minikube"

```
